server/api/inspirational_images_handler.py

```diff
@@ -0,0 +1,46 @@
+from flask import jsonify, Blueprint
+from api.models import InspirationalImage
+from api import db, s3
+from config import S3_BUCKET, S3_REGION
+from datetime import datetime
+import boto3
+import json
+
+inspirational_images_handler = Blueprint("inspirational_images_handler", __name__)
+
+@inspirational_images_handler.route('/inspirational_images_migration')
+def migrateImages():
+    s3_resource = boto3.resource('s3')
+    my_bucket = s3_resource.Bucket(S3_BUCKET)
+    summaries = my_bucket.objects.all()
```

Comment:
this is good, instead of grabbing all the objects in here, if it's possible, limit it to a folder just incase all their objects in that bucket is like 100 diff images 

if you limit so that developer has a folder called `/inspo_images` for example, you only grab the images from inside that folder